cephci/utils/configure.py

```diff
@@ -19,6 +22,18 @@
 
 CHMOD_CONFIG = f"chmod 600 {SSH_CONFIG}"
 SSHPASS = "sshpass -p {}"
+REGISTRY_AUTH_PATH = "/opt/registry/auth/htpasswd"
```

Comment:
Here `/opt/registry` is common path for most of the time. Use below format -

- Create variable for common format as `REGISTRY_PATH = "/opt/registry"`
- Use `REGISTRY_PATH` to create other variables like - `DOMAIN_KEY_PATH = f{REGISTRY_PATH}/certs/domain.key`